CONTRIBUTING.rst

```diff
@@ -181,6 +181,25 @@ related shells:
 .. _`cargo` : https://doc.rust-lang.org/cargo/
 .. _valgrind: https://valgrind.org/
 
+Inspecting code coverage
+------------------------
+
+The code coverage is reported at `codecov`_. You can also inspect the coverage locally.
+To get the full coverage first combine all reports and open produced html file in a
+browser
+
+.. code-block:: bash
+
+    tox
+    coverage combine \
+        ./python/metatensor-core/.coverage \
+        ./python/metatensor-torch/.coverage \
+        ./python/metatensor-operations/.coverage
+    coverage html
+    firefox htmlcov/index.html
```

Comment:
if we are suggesting this to users, we should add the created folders/files to gitignore

Reply:
Yes, good idea.